reana_client/cli/workflow.py

```diff
@@ -308,6 +308,46 @@ def workflow_start(ctx, workflow, access_token):
 def workflow_status(ctx, workflow, _filter, output_format,
                     access_token, verbose):
     """Get status of previously created workflow."""
+    def _get_data_from_row(row, data, headers):
+        name, run_number = get_workflow_name_and_run_number(
+            row['name'])
+        current_command = ''
+        current_step_name = ''
+        if 'current_command' in row['progress'] and \
+                row['progress'].get('current_command'):
+            if 'command' not in headers:
+                headers += ['command']
+            current_command = row['progress'].\
+                get('current_command')
+        if 'current_step_name' in row['progress'] and \
+                row['progress'].get('current_step_name'):
+            if 'step_name' not in headers:
+                headers += ['step']
+            current_step_name = row['progress'].\
+                get('current_step_name')
+        if current_command:
+            if current_command.startswith('bash -c "cd '):
+                current_command = current_command[
+                    current_command.index(';') + 2:-2]
+        else:
+            if 'command' in headers:
+                headers.remove('command')
+        data.append(list(map(
+            str,
+            [name,
+             run_number,
+             row['created'],
+             row['status'],
+             '{0}/{1}'.
+             format(
+                 row['progress']['succeeded'],
```

Comment:
Can you reproduce?
```console
$ export REANA_WORKON=$(reana-client create -f reana-yadage.yaml)
$ reana-client status
Workflow status could not be retrieved:
'succeeded'
```

Reply:
yes, I'm working on it I forgot to set the label